packages/sit-onyx/src/components/OnyxLoadingIndicator/LoadingDots.vue

```diff
@@ -1,43 +1,92 @@
 <template>
-  <svg class="onyx-loading-dots" viewBox="0 0 32 32">
-    <circle class="onyx-loading-dots__circle" cx="5" cy="16" r="3" />
-    <circle class="onyx-loading-dots__circle" cx="16" cy="16" r="2" />
-    <circle class="onyx-loading-dots__circle" cx="27" cy="16" r="3" />
-  </svg>
+  <div class="onyx-loading-dots">
+    <span class="onyx-loading-dots__center"></span>
+  </div>
 </template>
 
 <style lang="scss">
+/** Animation duration. */
+$duration: var(--onyx-duration-lg);
+
+/** Max size the dot should shrink in the animation. Must be a negative value. */
+$max-shrink: calc(-1 * var(--dot-size) / 2);
```

Comment:
is it intentionally that there is a moment where all the dots are invisible because their size shrinks to 0 (?)